Makefile

```diff
@@ -64,27 +67,88 @@ build-env:
 	    nx-X11/config/cf/nxversion.def.in \
 	    > nx-X11/config/cf/nxversion.def
 
+build-env: version
+	cd nx-X11 && make BuildIncludes FONT_DEFINES=$(FONT_DEFINES)
+
+	# set up environment for libNX_X11 build (X11 header files)
+	mkdir -p nx-X11/exports/include/nx-X11/
+	${SYMLINK_FILE} -t nx-X11/exports/include/nx-X11/ ../../../lib/include/X11/Xlib.h
```

Comment:
hmm, you run `cd nx-X11` in line 72, but then `mkdir nx-X11` _in that dir_. Looks wrong to me.